app/services/geocoder/update_all_schools.rb

```diff
@@ -0,0 +1,18 @@
+module Geocoder
+  class UpdateAllSchools
+    include ServicePattern
+
+    def call
+      Rails.logger.info "Updating schools with longitude and latitude!"
+
+      School.not_geocoded.find_each do |school|
+        school.geocode
+        school.save!
+      rescue ActiveRecord::RecordInvalid => e
+        Sentry.capture_exception(e)
+      end
+
+      Rails.logger.info "All schools updated with longitude and latitude!"
```

Comment:
Could this be misleading if there are errors?

Reply:
Resolved in [c33d97b](https://github.com/DFE-Digital/itt-mentor-services/pull/382/commits/c33d97b1f77a5bbdd5a6212b8e5e6bff47a6dd8d)